app.json

```
@@ -124,6 +124,14 @@
       "description": "Value provided by Heroku containing the parent app name (eg micromasters-ci for a PR build)",
       "required": true
     },
+    "HUBSPOT_NEW_COURSES_FORM_GUID": {
+      "description": "Form guid over hub spot for new courses email subscription form.",
+      "required": false
+    },
```

```suggestion
    "HUBSPOT_ORGANIZATIONS_FORM_GUID": {
      "description": "Hubspot guid for Organizations contact form on /organizations/ page",
      "required": false
    },
```

We can make this more specific to the use case on MicroMasters